src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -288,6 +288,12 @@ configurationRegistry.registerConfiguration({
 			'type': 'boolean',
 			'description': nls.localize('enableNaturalLanguageSettingsSearch', "Controls whether to enable the natural language search mode for settings."),
 			'default': true
+		},
+		// {{SQL CARBON EDIT}}
+		'workbench.editor.showConnectionInfoInTitle': {
```

Comment:
it is possible to add this setting from under the `sql` directory somewhere?  I'd refer to keep the `vs` changes minimized as much as possible, particularly for SQL-related changes like this that we wouldn't consider pushing upstream at some point.

Reply:
I was thinking from a user point of view, i'm not going to know the different between sql settings and non-sql settings, so putting a tab title settings near all the other tab title settings made the most sense from a user perspective.